src/pybind/mgr/dashboard/frontend/src/app/ceph/block/rbd-snapshot-form/rbd-snapshot-form-modal.component.html

```diff
@@ -37,6 +37,9 @@
           <span *ngIf="snapshotForm.showError('snapshotName', formDir, 'required')"
                 class="invalid-feedback"
                 i18n>This field is required.</span>
+          <span *ngIf="snapshotForm.controls['snapshotName'].hasError('invalidCharacters')"
+                class="invalid-feedback"
+                i18n>The snapshot name cannot be "." or "/".</span>
```

Comment:
```suggestion
                i18n>The snapshot name cannot start with "." or "/".</span>
```

Reply:
This was correct before, right @dnyanee1997?
As the regex was validating to check no snapshot name could be either `.` or `/`

We also make mistakes while reviewing, and no one knows better your changes than you so it's completely fine to argue against any proposed changed

Reply:
Yup! Just reverted the code. I think now it will be fine.